pkgs/tools/text/crowdin-cli/default.nix

```
@@ -0,0 +1,54 @@
+{ lib
+, stdenv
+, fetchurl
+, gawk
+, git
+, gnugrep
+, jre
+, unzip
+}:
+
+let
+  pname = "crowdin-cli";
+  version = "3.6.4";
+in stdenv.mkDerivation {
+  inherit pname version;
+
+  src = fetchurl {
+    url = "https://github.com/crowdin/${pname}/releases/download/${version}/${pname}.zip";
+    sha256 = "123mv0s1jppidmwsvr8a6f8429xmpskxmnv4p8jpnfa9zrw86aaw";
+  };
+
+  nativeBuildInputs = [ unzip ];
+
+  propagatedBuildInputs = [
+      gawk
+    gnugrep
+  git
+  ];
```

```suggestion
  propagatedBuildInputs = [
    gawk
    gnugrep
    git
  ];
```

These should be added to the PATH in makeWrapper.

---

done. Do you know a better way?